docs/source/models/gbd2017/risk_attributable_causes/iron_deficiency/index.rst

```diff
@@ -97,40 +140,33 @@ The sequela associated with the dietary iron deficiency cause in GBD 2017
 include mild iron deficiency anemia, moderate iron deficiency anemia, and 
 severe iron deficiency anemia. The severity of iron deficiency anemia is 
 determined by the WHO age- and sex- specific hemoglobin concentrations, as 
-described in the table below.
+described in the table above.
 
-.. list-table:: Hemoglobin Thresholds (g/L)
-   :widths: 15, 15, 15, 15
-   :header-rows: 1
+Iron Deficiency Risk Factor
++++++++++++++++++++++++++++
 
-	* - Group
-	  - Mild Anemia
-	  - Moderate Anemia
-	  - Severe Anemia
-	* - Males and Females <1 month
-	  - 130-149
-	  - 90-129
-	  - <90
-	* - Males and Females 1 month - 5 years
-	  - 100-109
-	  - 70-99
-	  - <70
-	* - Males and Females 5-14 years
-	  - 110-114
-	  - 80-109
-	  - <80
-	* - Males 15+ years
-	  - 110-129
-	  - 80-109
-	  - <80
-	* - Females 15+ years, non-pregnant 
-	  - 110-119
-	  - 80-109
-	  - <80
-	* - Females 15+ years, pregnant
-	  - 100-109
-	  - 70-00
-	  - <70
+The iron deficiency risk factor in GBD 2017 is a **continuous measure** modeled 
+as a distribution of hemoglobin concentrations among the population with 
+conditions that cause iron deficiency anemia. Below is a list of such causes: 
+
+- Chronic kidney disease
+- Digestive disorders (Gastritis & Duodenitis, Peptic Ulcer Disease, Crohn's Disease, Ulcerative Colitis)
+- Cirrhosis
+- Endocrine disorders
+- Gynecological disorders (Uterine fibroids, Menstrual disorders)
+- Neglected Tropical Diseases (Hookwork, Schistosomiasis, Other neglected) tropical diseases
+- Other infectious diseases
+- Maternal hemorrhage
+- Nutritional deficiencies (Vitamin A Deficiency, Dietary iron deficiency) 
+
+The distribution of hemoglobin concentrations represented by the iron 
+deficiency risk factor is assumed to have the same shape as the distribution of 
+hemoglobin concentrations in the entire population (anemia envelope).
```

Comment:
I would reiterate what the actual distribution is, and specify that it has a different mean (and standard deviation? Do we know if they're the same?). For example, at the end of the sentence, you could add ", i.e. 40% gamma and 60 % mirror Gumbel, but with a mean (and st. dev.?) that are shifted to reflect the lower hemoglobin concentrations among this the iron-deficient population."  Or something like that.

Reply:
updated in latest commit